doc-content/enterprise-only/installation/clustering-dev-con.adoc

```diff
@@ -19,12 +19,14 @@ endif::[]
 
 To create a {PRODUCT} clustered development environment, you must perform the following tasks:
 
-* Configure {EAP} {EAP_VERSION} with {DATAGRID} {DATAGRID_VERSION} on a machine.
-* Configure AMQ Broker, a Java messaging server (JMS) broker, on a machine.
-* Configure an NFS file server on a machine.
-* Download {EAP} {EAP_VERSION} and {PRODUCT} {PRODUCT_VERSION}, then install them on each machine that is to become a cluster node.
-* Configure and start {CENTRAL} on each cluster node to start the operation of the cluster.
+. Configure the following components on a system that will be a node of the cluster:
```

Comment:
[Style} wdyt about reintroducing "each"?

s/Configure the following components on a system that will be a node of the cluster:/Configure the following components on each system that will be a node of the cluster/Configure the following components on each node of the cluster

Reply:
Yes. Good catch.